python/usb_console/radio_session.py

```diff
@@ -36,8 +41,16 @@ def __init__(self, device_name, datastore, logger):
         self.datastore = datastore
         self.logger = logger
 
-        # Simulation
-        self.overriden_variables = set()
+        #email        
+        with open(os.path.join(os.path.dirname(os.path.abspath(__file__)), 'configs/confidential.json')) as config_file:
+            data = json.load(config_file)
+        self.username=data["PANUsername"]
```

Comment:
Same deal here with description: I think it would be better if this is called the "email_username" and "email_password".